cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Query.hs

```diff
@@ -82,41 +44,3 @@ resolveInputTxOutIdValue txIn =
 queryResolveInputCredentials :: MonadIO m => Generic.TxIn -> ReaderT SqlBackend m (Either LookupFail (Maybe ByteString, Bool))
 queryResolveInputCredentials txIn = do
   queryTxOutCredentials (Generic.txInHash txIn, fromIntegral (Generic.txInIndex txIn))
-
-queryStakeRefPtr :: MonadIO m => Ptr -> ReaderT SqlBackend m (Maybe StakeAddressId)
-queryStakeRefPtr (Ptr (SlotNo slot) (TxIx txIx) (CertIx certIx)) = do
-  res <- select $ do
-    (blk :& tx :& sr) <-
-      from
-        $ table @Block
-          `innerJoin` table @Tx
-        `on` (\(blk :& tx) -> blk ^. BlockId ==. tx ^. TxBlockId)
-          `innerJoin` table @StakeRegistration
-        `on` (\(_blk :& tx :& sr) -> sr ^. StakeRegistrationTxId ==. tx ^. TxId)
-
-    where_ (blk ^. BlockSlotNo ==. just (val slot))
-    where_ (tx ^. TxBlockIndex ==. val (fromIntegral txIx))
-    where_ (sr ^. StakeRegistrationCertIndex ==. val (fromIntegral certIx))
-    -- Need to order by DelegationSlotNo descending for correct behavior when there are two
-    -- or more delegation certificates in a single epoch.
-    orderBy [desc (blk ^. BlockSlotNo)]
-    limit 1
-    pure (sr ^. StakeRegistrationAddrId)
-  pure $ unValue <$> listToMaybe res
-
--- Check if there are other PoolUpdates in the same blocks for the same pool
-queryPoolUpdateByBlock :: MonadIO m => BlockId -> PoolHashId -> ReaderT SqlBackend m Bool
-queryPoolUpdateByBlock blkId poolHashId = do
```

Comment:
👌 I did find it confusing that we had db queries inside of cardano-db-sync but was too scared to move 😅